docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -111,58 +116,65 @@ To speed up the time it takes to review your plugin:
-   - The remaining questions help us determine the [signature level](https://grafana.com/docs/grafana/latest/plugins/plugin-signatures/#plugin-signature-levels) for your plugin.
+   - The remaining questions help us determine the [signature level]({{< relref "./sign-a-plugin#plugin-signature-levels" >}}) for your plugin.
 1. Click **Submit**.
-   After you have submitted your plugin, we run an automated validation to make sure it adheres to our guidelines. Once your submission passes the validation, it's placed in a review queue.
+   After you have submitted your plugin, we run an automated validation to make sure it adheres to our guidelines.
+   Once your submission passes the validation, it's placed in a review queue.
 
-All submissions are manually inspected by a plugin reviewer. For every new plugin, we perform a manual review that includes the following checks:
+All submissions are manually inspected by a plugin reviewer.
+For every new plugin, we perform a manual review that includes the following checks:
 
-- **Code review:** For quality and security purposes, we review the source code for the plugin. If you're unable to make the source code publicly available, let us know in a comment on your plugin submission.
-- **Tests:** We install your plugin on one of our Grafana instances to test it for basic use. For more advanced plugins, we may ask you to assist us in configuring a test environment for the plugin. This test environment will be used whenever you submit an plugin update.
+- **Code review:** For quality and security purposes, we review the source code for the plugin.
+  If you're unable to make the source code publicly available, let us know in a comment on your plugin submission.
+- **Tests:** We install your plugin on one of our Grafana instances to test it for basic use.
+  For more advanced plugins, we may ask you to assist us in configuring a test environment for the plugin.
+  This test environment will be used whenever you submit an plugin update.
 
-## Submit a plugin update
+### Submit a plugin update
 
 To submit an **update** for an already published plugin:
 
-1. [Sign in](https://grafana.com/auth/sign-in) to your Grafana Cloud account.
+1. [Sign in](/auth/sign-in) to your Grafana Cloud account.
 1. In the left menu, under **Org settings**, click **My Plugins**.
 1. Click **Submit Update** for the plugin you want to update.
 1. Enter the information requested by the form.
-   - **OS & Architecture:**
-     - Select **Single** if your plugin archive contains binaries for multiple architectures.
-     - Select **Multiple** if you'd like to submit separate plugin archives for each architecture. This can lead to faster downloads since users can select the specific architecture they want to install the plugin on.
+   - **OS & Architecture:** - Select **Single** if your plugin archive contains binaries for multiple architectures. - Select **Multiple** if you'd like to submit separate plugin archives for each architecture.
```

Comment:
Is this no longer a nested checklist? - Both `Select **Single**..`and `Select **Multiple**..`